qlty-check/src/executor.rs

```diff
@@ -557,10 +576,10 @@ fn run_invocation(
     transformers: &[Box<dyn IssueTransformer>],
 ) -> Result<InvocationResult> {
     let task = progress.task(&plan.plugin_name, &plan.description());
-    let mut invocation = plan.driver.run(&plan, &task)?;
+    let mut result = plan.driver.run(&plan, &task)?;
```

Comment:
This is actually an `InvocationResult` not an `Invocation` so I adjusted this local variable name for clarity.